dali/base/dadfs.cpp

```diff
@@ -5225,12 +5226,38 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
         init(_parent,tree,_name,user,transaction);
     }
 
+    CDistributedSuperFile(CDistributedFileDirectory *_parent, ISuperFileDescriptor *_desc, IUserDescriptor* user)
+    {
+        parent = _parent;
+        setUserDescriptor(udesc, user);
+        interleaved=0;
+        root.set(_desc->getProperties());
+        const char *val = root->queryProp("@interleaved");
+        if (val)
+        {
+            if (isdigit(*val))
+                interleaved = atoi(val);
+            else
+                interleaved = strToBool(val)?1:0;
+        }
+    }
+
     ~CDistributedSuperFile()
     {
         partscache.kill();
         subfiles.kill();
     }
 
+    void relinkSubFile(unsigned pos)
+    {
+        linkSubFile(pos);
+    }
+
+    void reloadSubFiles()
+    {
+        loadSubFiles(NULL,INFINITE);
+    }
```

Comment:
Shouldn't need these, work should be done in a CDistributedSuperFile constructor.